### DISCUSSION TITLE
Open source seismic tomography libraries

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#201] [General]
Hi,

I'm looking for methods for velocity model building/preparation for FWI. Especially I'm interested in testing refraction-reflection tomography opportunities.

It would be great if I could test it on Viking Graben open source data before doing FWI.

I thought maybe you know open source projects with such functionality? 
I tried [pyGIMLI](https://www.pygimli.org/) but unfortunately it only works with first breaks.

**Comment by mloubout** [comment]
I'm not aware of any. I'm sure there must be something out there in python or Julia but we never really looked into it.
